src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/InstanceAppSI.cs

```diff
@@ -218,6 +218,24 @@ public async Task<Instance> UpdateSubstatus(int instanceOwnerPartyId, Guid insta
             throw await PlatformHttpException.CreateAsync(response);
         }
 
+        /// <inheritdoc />
+        public async Task<Instance> UpdatePresentationTexts(int instanceOwnerPartyId, Guid instanceGuid, PresentationTexts presentationTexts)
```

Comment:
Could it be an idea to combine the two parameters in one, instanceId? This isn't the only method where would could change this, so maybe a separate issue? You decide.

Reply:
Made a seperate issue so we can cover all the platform services #6107